src/conf.py

```diff
@@ -53,7 +53,6 @@
                   "coreComponents/dataRepository",
                   "coreComponents/fileIO",
                   "coreComponents/linearAlgebra",
-                  "coreComponents/LvArray/src",
```

Comment:
So how is the `LvArray` doc build triggered if `LvArray` is no more considered as a submodule of `GEOSX`.
There is a read the docs build for `LvArray` as well?

Reply:
> So how is the `LvArray` doc build triggered if `LvArray` is no more considered as a submodule of `GEOSX`. There is a read the docs build for `LvArray` as well?

Yes, LvArray has its own ReadTheDocs page.